Library/Homebrew/.rubocop.yml

```diff
@@ -72,6 +77,11 @@ Naming/PredicateName:
 Naming/UncommunicativeMethodParamName:
   Enabled: false
 
+# I'm not sure how to correct these, and seems to get false positives on
+# modifiers used on symbols of methods
```

Comment:
Change this comment to reference https://github.com/rubocop-hq/rubocop/issues/5953.

Reply:
Done.